dev/io.openliberty.org.jboss.resteasy.common.ee10/src/org/jboss/resteasy/plugins/server/servlet/ServletUtil.java

```diff
@@ -0,0 +1,204 @@
+package org.jboss.resteasy.plugins.server.servlet;
+
+import org.jboss.resteasy.core.Headers;
+import org.jboss.resteasy.plugins.server.servlet.ServletUtil.InitDataCacheKey;
+import org.jboss.resteasy.specimpl.ResteasyHttpHeaders;
+import org.jboss.resteasy.specimpl.ResteasyUriInfo;
+import org.jboss.resteasy.specimpl.ResteasyUriInfo.InitData;
+import org.jboss.resteasy.util.HttpHeaderNames;
+import org.jboss.resteasy.util.MediaTypeHelper;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.ws.rs.core.Cookie;
+import jakarta.ws.rs.core.HttpHeaders;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.MultivaluedMap;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+/**
+ * @author <a href="mailto:bill@burkecentral.com">Bill Burke</a>
+ * @version $Revision: 1 $
+ */
+public class ServletUtil
+{
+   private static Map<InitDataCacheKey, InitData> initDataCache = new ConcurrentHashMap<InitDataCacheKey, InitData>();      // Liberty change
```

Comment:
```suggestion
   private static final Map<InitDataCacheKey, InitData> initDataCache = new ConcurrentHashMap<InitDataCacheKey, InitData>();      // Liberty change
```

Reply:
Thanks for the feedback. I think I've made all these changes in both copies of the class.